configs/katello_33.yaml

```diff
@@ -0,0 +1,178 @@
+        :katello-3.3-rhel7: 0
+      :katello-3.3-rhel7:
+        :katello-3.3-thirdparty-rhel7: 0
+      :katello-3.3-client-rhel7:
+        :katello-3.3-rhel7: 0
+        :katello-3.3-thirdparty-pulp-rhel7: 2
+    :external_repos:
+      - rhel-7.0-server
+      - rhel-7.0-server-optional
+      - epel-7
+      - centos-sclo-rh-rhel-7
+      - centos-sclo-sclo-rhel-7
+  - :name: katello-3.3-rhel6
+    :based_off: katello-nightly-rhel6
+    :helper_tags:
+      :katello-3.3-thirdparty-rhel6: katello-nightly-thirdparty-rhel6
+      :katello-3.3-rhel6-override: null
+      :katello-3.3-client-rhel6: null
+    :build_target: katello-3.3-rhel6-build
+    :build_package_group_source_tag: katello-nightly-rhel6-build
+    :arches:
+      - x86_64
+    :inherits:
+      :katello-3.3-rhel6-build:
+        :katello-3.3-rhel6-override: 0
+      :katello-3.3-rhel6-override:
+        :katello-3.3-rhel6: 1
+      :katello-3.3-rhel6:
+        :katello-3.3-thirdparty-rhel6: 0
+      :katello-3.3-client-rhel6:
+        :katello-3.3-rhel6: 0
+        :katello-3.3-thirdparty-pulp-rhel6: 2
+    :external_repos:
+      - rhel-6.6-server
+      - rhel-6.6-server-optional
+      - epel-6
+  - :name: katello-3.3-rhel5
+    :based_off: katello-nightly-rhel5
+    :helper_tags:
+      :katello-3.3-rhel5-override: null
+      :katello-3.3-client-rhel5: null
+    :build_target: katello-3.3-rhel5-build
+    :build_package_group_source_tag: katello-nightly-rhel5-build
+    :arches:
+      - x86_64
+    :inherits:
+      :katello-3.3-rhel5-build:
+        :katello-3.3-rhel5-override: 0
+      :katello-3.3-rhel5-override:
+        :katello-3.3-rhel5: 0
+      :katello-3.3-rhel5: {}
+      :katello-3.3-client-rhel5:
+        :katello-3.3-rhel5: 0
+        :katello-3.3-thirdparty-pulp-rhel5: 2
+    :external_repos:
+      - rhel-5.9-server
+      - epel-5
+  - :name: katello-3.3-fedora24
+    :based_off: katello-nightly-fedora24
+    :helper_tags:
+      :katello-3.3-client-fedora24: null
+    :build_target: katello-3.3-fedora24-build
+    :build_package_group_source_tag: katello-nightly-fedora24-build
+    :arches:
+      - x86_64
+    :inherits:
+      :katello-3.3-fedora24-build:
+        :katello-3.3-fedora24: 0
+      :katello-3.3-fedora24: {}
+      :katello-3.3client-fedora24:
+        :katello-3.3-fedora24: 0
+        :katello-3.3-thirdparty-pulp-fedora24: 2
+    :external_repos:
+      - fedora-24-updates
+      - fedora-24
+  - :name: katello-3.3-fedora25
+    :based_off: katello-nightly-fedora25
+    :helper_tags:
+      :katello-3.3-client-fedora25: null
+    :build_target: katello-3.3-fedora25-build
+    :build_package_group_source_tag: katello-nightly-fedora24-build
+    :arches:
+      - x86_64
+    :inherits:
+      :katello-3.3-fedora25-build:
+        :katello-3.3-fedora25: 0
+      :katello-3.3-fedora25: {}
+      :katello-3.3-client-fedora25:
+        :katello-3.3-fedora25: 0
+    :external_repos:
+      - fedora-25-updates
+      - fedora-25
+
+
+
+
```

Comment:
you caught me, i've been embedding programs in my [white space](https://en.wikipedia.org/wiki/Whitespace_(programming_language))